tests/stream/stream_constructors.cpp

```diff
@@ -80,6 +80,27 @@ class TEST_NAME : public util::test_base {
         });
       }
 
+      /** check (size_t, size_t, sycl::handler&, sycl::property_list&)
+       * constructor
+       */
+      {
+        queue.submit([&](sycl::handler &handler) {
+          sycl::property_list property_list{};
+          sycl::stream os(bufferSize, maxStatementSize, handler, property_list);
+
+          if (os.get_size() != bufferSize) {
```

Comment:
`get_size` is deprecated. Please, use `size` instead.

This must be applied to all other stream test cases.

Reply:
Fixed, also by replacing `get_max_statement_size` with `get_work_item_buffer_size`.

Reply:
An unfortunate side effect is that the constructor test must now be disabled for DPCPP, as it does not yet implement the SYCL2020 API here.